frontends/web/src/declaration.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.css" {
+    const mapping: Record<string, string>;
+    export default mapping;
+}
```

Comment:
Prevents an error when importing a .css file that has no .css.d.ts file. The linter will still error with

"Property 'prettyButton' does not exist on type 'typeof import("*.css")'.ts(2339)"